### ISSUE TITLE
SISNR implementation

### ISSUE THREAD

**Issue by krantiparida** [haoheliu/voicefixer_main#14]
Hi, Thanks for the awesome work. I was going through the Si-SNR implementation but found that you have used negative of sdr [here](https://github.com/haoheliu/voicefixer_main/blob/3c0049c5b5be29d4cb92c0a8cf69de8c19be4f47/tools/pytorch/metrics/sisnr.py#L54) . If I have not grossly misunderstood anything, can you please let me know why you have used negative here? Thanks .

**Comment by haoheliu** [comment]
Hi Kranti,

Thanks for the interest of this repo. I add the negative sign in the Si-SNR because I once tried to use it as loss function. The negative sign turn loss minimization into Si-SNR maxmization. Hope this helps.

Haohe

**Comment by krantiparida** [comment]
Thanks for the prompt response. So, as I understand, if I need to evaluate using the implementation of Si-SNR then I have to remove the negative sign?

**Comment by haoheliu** [comment]
Exactly, you need to remove the negative sign if you use it for evaluation.

**Comment by krantiparida** [comment]
Thanks for the clarification.
